### ISSUE TITLE
Add a tail all deployments, statefulset, daemonset option

### ISSUE THREAD

**Issue by jkroepke** [kubetail-org/kubetail#25]
It comes from the context of namespaces deployment.

I like the idea to have an URL which opens kubetail directly. I saw this behavior for the demo here: https://www.kubetail.com/demo

I would like to offer an direct link which open all pods from the current namespace. Currently, I have to known all names of the deployments and put them inside the url.

**Comment by amorey** [comment]
Makes sense. The app has access to all the data necessary to implement this so we would just need to add some UI features to enable it. Right now I'm focused on solving some bottlenecks in the UI (https://github.com/kubetail-org/kubetail/issues/2) but that should be done soon and then I can estimate how long it would take to implement this.

**Comment by amorey** [comment]
@jkroepke Would this url structure work for you?

```
/console?source=<namespace>/<workload-type>/<workload-name>
```

Where `<workload-name>` can be a wildcard (`*`)? For example, this would tail all the pods in the "default" and "kube-system" namespaces:

```
/console?source=default%2Fpods%2F/*&source=kube-system%2Fpods%2F*
```

**Comment by jkroepke** [comment]
Yes, that would be great

**Comment by amorey** [comment]
Thanks, now you can use a wildcard to tail all pods in a given namespace:
https://demo.kubetail.com/console?source=kubetail-demo%2Fpods%2F*

I left this undocumented for now and you can't add/remove it in the "Sources" list like you can with other sources. Let me know how you're finding the feature and once it's more stable I can work on integrating it into the UI.
